xblockutils/public/studio_edit.js

```diff
@@ -65,6 +66,11 @@ function StudioEditableXBlockMixin(runtime, element) {
                 }
             });
         }
+
+        if (type == 'datepicker' && datepickerAvailable) {
+            $field.datepicker('destroy');
+            $field.datepicker({dateFormat: "m/d/yy"});
```

Comment:
@tusbar @bradenmacdonald I was actually suggesting making ISO 8601 world-wide locale to stop this kind of discussions once and for all. Roughly the same thing as switching to 100-second minute and making US actually use metric system :)